modules/diffusion_solver/diffusion_solver.h

```diff
@@ -39,6 +33,15 @@ class Solver : public opensn::Solver
   double time_solve_ = 0.0;
   bool verbose_info_ = true;
 
+  enum MaterialMode
+  {
+    DIFFUSION_MATERIALS_REGULAR = 10,
+    DIFFUSION_MATERIALS_FROM_TRANSPORTXS_TTR = 11,
+    DIFFUSION_MATERIALS_FROM_TRANSPORTXS_TTF = 12,
+    DIFFUSION_MATERIALS_FROM_TRANSPORTXS_TTF_JPART = 13,
+    DIFFUSION_MATERIALS_FROM_TRANSPORTXS_TTF_JFULL = 14
+  };
```

Comment:
While the names are old, since this is in a diffusion solver and now under the `MaterialMode` enum, how about removing the `DIFFUSION_MATERIALS` prefixes?

Reply:
:+1: Change made.